simpleflow/command.py

```diff
@@ -472,19 +465,13 @@ def start_worker(
     nb_processes,
     heartbeat,
     one_task,
-    process_mode,
     poll_data,
     middleware_pre_execution,
     middleware_post_execution,
 ):
     if log_level:
         logger.warning("Deprecated: --log-level will be removed, use LOG_LEVEL environment variable instead")
 
-    if process_mode == "kubernetes" and poll_data:
-        # don't accept to have a worker that doesn't poll AND doesn't process
-        # since it would be just a gate to a scheduling infinite loop
-        raise ValueError("--process-mode=kubernetes and --poll-data options are exclusive")
```

Comment:
It's a bit late now but just so you know, there's an [option](https://docs.python.org/3/library/argparse.html#mutual-exclusion) for making flags mutually exclusive in argparse